python/ray/ml/preprocessors/encoder.py

```diff
@@ -15,33 +17,58 @@ class OrdinalEncoder(Preprocessor):
 
     Transforming values not included in the fitted dataset will be encoded as ``None``.
 
+    All column values must be hashable scalars or lists of hashable values.
```

Comment:
Should we call out the assumption here? A column can be either all scalars or all lists. Cannot be a mix?

Reply:
No, it cannot. I am not sure if arrow even allows that.